open-sphere-base/core/src/main/resources/styles/opensphere.css

```diff
@@ -27,6 +27,18 @@
     -fx-text-fill: linear-gradient(to bottom, derive(-opensphere-dark, 20%) 10%, derive(-opensphere-dark, -40%) 80%);
 }
 
+.icons-FontAwesome {
+    -fx-font-family: FontAwesome !important;
```

Comment:
What rule is !important supposed to be overriding? This css seems pretty specific to each class and shouldnt need the !important tag

Reply:
They're applied to the same items as the `.icons` class.